springwolf-examples/springwolf-cloud-stream-example/src/main/resources/application.properties

```diff
@@ -39,7 +42,11 @@ springwolf.docket.info.contact.url=https://github.com/springwolf/springwolf-core
 springwolf.docket.info.license.name=Apache License 2.0
 springwolf.docket.servers.kafka-server.protocol=kafka
 springwolf.docket.servers.kafka-server.host=${spring.cloud.stream.binders.kafka.environment.spring.kafka.bootstrap-servers}
-springwolf.use-fqn=false
```

Comment:
Originally, we used cloud-stream to demo (and integration test) use-fqn=false,
but it could be the time to switch this.
I opened https://github.com/springwolf/springwolf-core/pull/1078 for a dedicated test to keep the test coverage

Reply:
I wasn't able to publish without this, because of `ClassNotFoundException`s

Reply:
Fair. You can combine the cloud-stream and kafka plugin in your project for it.

I guess the project setup is not documented well enough.
Each example projects allows us to test/verify that its matching plugin works as expected.

Unfortunately, `core` has limited spring boot integration test capabilities - something we are working on currently. Therefore, the `example` projects serve as integration tests as well and are independent of each other.